test/e2e/bmc.go

```diff
@@ -0,0 +1,27 @@
+package e2e
+
+import (
+	"os"
+
+	. "github.com/onsi/gomega"
+	"gopkg.in/yaml.v2"
+)
+
+// BMC defines a BMH to use in the tests
+type BMC struct {
+	User           string `yaml:"user,omitempty"`
+	Password       string `yaml:"password,omitempty"`
+	Address        string `yaml:"address,omitempty"`
+	BootMacAddress string `yaml:"bootMacAddress,omitempty"`
+	HostName       string `yaml:"hostName,omitempty"`
+	IpAddress      string `yaml:"ipAddress,omitempty"`
+	SSHPort        string `yaml:"sshPort,omitempty"`
```

Comment:
Could you add doc strings for all of these? I think especially the IP address and ssh port we should point out that they are optional and used only for the ssh check.

Reply:
Sure!